src/py-opentimelineio/opentimelineio/adapters/adapter.py

```diff
@@ -40,6 +40,13 @@
 )
 
 
+try:
+    # Python 3.0+
+    getargspec = inspect.getfullargspec
+except AttributeError:
+    getargspec = inspect.getargspec
```

Comment:
do we have a consistent way of labelling hacks like this?  Just something to make it easier to find these later when we eventually drop python 2 support.

Also, I wonder if it should follow the newer convention rather than older one.  In other words:

`inspect_getfullargspec = ...` rather than `getargspec = ...`

Reply:
I can take a quick peek at turning on Python 3.9 for our CI.
I agree with your note about using the new `getfullargspec`, will update for that.
In terms of consistent ways of labeling the hack, I think implementing #821 would help for a lot of these, though this particular one doesn't seem to be in the cases it addresses.